app/src/main/java/org/jellyfin/androidtv/ui/itemhandling/BaseRowItem.kt

```diff
@@ -152,27 +152,27 @@ open class BaseRowItem protected constructor(
 		BaseItemKind.MUSIC_ARTIST
 	).contains(baseItem?.type)
 
-	fun getImageUrl(context: Context, imageType: ImageType, maxHeight: Int) = when (baseRowType) {
+	fun getImageUrl(context: Context, imageType: ImageType, preferredWidth: Int) = when (baseRowType) {
 		BaseRowType.BaseItem,
 		BaseRowType.LiveTvProgram,
 		BaseRowType.LiveTvRecording -> {
 			val apiClient = get<LegacyApiClient>()
 			when (imageType) {
-				ImageType.BANNER -> ImageUtils.getBannerImageUrl(baseItem, apiClient, maxHeight)
-				ImageType.THUMB -> ImageUtils.getThumbImageUrl(baseItem, apiClient, maxHeight)
-				else -> getPrimaryImageUrl(context, maxHeight)
+				ImageType.BANNER -> ImageUtils.getBannerImageUrl(baseItem, apiClient, preferredWidth)
+				ImageType.THUMB -> ImageUtils.getThumbImageUrl(baseItem, apiClient, preferredWidth)
```

Comment:
We should only use the preferredWidth for the banner/thumb (as their aspect ratio is above 1), for the poster we should keep the preferredHeight. Also make sure only those two use the preferredWidth so we don't introduce changes for other places that use images, as its quite hard to check every part of the app for regression.

Reply:
Wouldn't the first commit be enough to fix the issue then? As it's a simple one-liner that calculates the required height of the image needed to fill the banner, it's entirely self-contained and the only shortcoming it has is that it doesn't call maxWidth by name, which I don't think matters as long as the results are the same...

Reply:
No, the first commit still relies on sending a max height and hacks that by multiplying the requested value

Reply:
Right, but it's not multiplied randomly. It calculates the exact size of the poster image required to fill the card based on the aspect ratios.